concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1267,17 +1265,14 @@ jobs:
 
                   ./paas-bootstrap/concourse/scripts/fly_sync_and_login.sh
                   # shellcheck disable=SC2046,SC2116
-                  ${FLY_CMD} -t "${FLY_TARGET}" set-team -n "${FLY_TEAM}" \
+                  ${FLY_CMD} -t "${FLY_TARGET}" set-team -n main \
                     --github-auth-client-id "${GITHUB_CLIENT_ID}" \
                     --github-auth-client-secret "${GITHUB_CLIENT_SECRET}"  \
                     $(echo "$GITHUB_USERS") --basic-auth-username admin --basic-auth-password "${CONCOURSE_ATC_PASSWORD}" --non-interactive
 
                   ./paas-bootstrap/concourse/scripts/fly_sync_and_login.sh
-                  # shellcheck disable=SC2046,SC2116
-                  ${FLY_CMD} -t "${FLY_TARGET}" set-team -n main \
-                    --github-auth-client-id "${GITHUB_CLIENT_ID}" \
-                    --github-auth-client-secret "${GITHUB_CLIENT_SECRET}"  \
-                    $(echo "$GITHUB_USERS") --basic-auth-username admin --basic-auth-password "${CONCOURSE_ATC_PASSWORD}" --non-interactive
+                  ${FLY_CMD} -t "${FLY_TARGET}" set-team -n "${FLY_TEAM}" \
+                    --basic-auth-username admin --basic-auth-password "${CONCOURSE_ATC_PASSWORD}" --non-interactive
```

Comment:
I'm confused by this new version. It seems to duplicate the lines below...

I'm also not clear why we're calling `fly_sync_and_login` so many times.

As I think about this more, I don't think the oauth stuff belongs in this task at all - it's got nothing to do with creating the env-specific team. I think we should create a new task to setup the oauth stuff.

Reply:
https://github.com/alphagov/paas-bootstrap/blob/b6b56271879211f95886db6938a92759daba3ad7/concourse/pipelines/create-bosh-concourse.yml#L1280-L1282 is the action performed if ENABLE_GITHUB=false

We could break it into a separate task if desired.